layer/tracker/ui_progress_tracker.py

```diff
@@ -301,7 +304,7 @@ def mark_model_trained(
         self._update_asset(
             AssetType.MODEL,
             name,
-            status=AssetTrackerStatus.DONE,
+            status=AssetTrackerStatus.TRAINING,
```

Comment:
Are you sure this is right? The function name is `mark_model_trained` but we're marking it as `training`.